doc/source/whatsnew/v1.0.0.rst

```diff
@@ -229,6 +229,7 @@ Indexing
 - Bug in reindexing a :meth:`PeriodIndex` with another type of index that contained a `Period` (:issue:`28323`) (:issue:`28337`)
 - Fix assignment of column via `.loc` with numpy non-ns datetime type (:issue:`27395`)
 - Bug in :meth:`Float64Index.astype` where ``np.inf`` was not handled properly when casting to an integer dtype (:issue:`28475`)
+- Bug in :meth:`MultiIndex.get_loc` can't find ``np.nan`` with a null value as input (:issue:`19132`)
```

Comment:
I would mention "partial indexing" and "first level" somewhere - assuming this is really specific to first level.

Reply:
When 'get_loc' argument is just one, 'get_loc' in 'MultiIndex' searches at first level before i send PR. I think there was an implicit agreement, so i didn't put in whatsnew.

Reply:
I would have expected a fix to #19132 to also solve the case ``mi.get_loc((1.0, np.nan))``. This PR doesn't, fair enough, but I would make it slightly more explicit. Just "can't find ``np.nan`` in first level" is probably enough.

Reply:
@toobaz. I missunderstood issue. I thought just want to fix with a one nan. Okay. It's my fault. I want to fix completely. I try to figure out and fix all.

Reply:
OK, as you prefer. I'm not saying the aim of this PR was not good. Then sure, if you are able to provide a more general solution, that's great.

Reply:
@toobaz Thank you for your review and pointing out sharply